packages/venia-ui/lib/components/CartPage/cartPage.js

```diff
@@ -37,11 +42,11 @@ const CartPage = props => {
                     <ProductListing />
                 </div>
                 <div className={classes.price_adjustments_container}>
-                    <PriceAdjustments />
+                    {hasItems ? <PriceAdjustments /> : null}
```

Comment:
Should we do the same here for `ProductListing`? That component (and `PriceSummary`, actually) have a `null` render state based on the data they receive back.

If we add a conditional before rendering though, we prevent the component from mounting/making a graphQL query, which could be a win. Thoughts?